test/show.jl

```diff
@@ -1387,6 +1387,18 @@ end
 
     d = Dict("+"=>1)
     @test showstr(d) == "Dict(\"+\" => 1)"
+
+    struct Foo
+        a::Int
+    end
+    struct Bar
+        a::Int
+    end
+    d = Dict([Bar(1), Bar(2)] => [Foo(1), Foo(2)])
+    @test showstr(d) == "Dict{Vector{$(curmod_prefix)Bar}, Vector{$(curmod_prefix)Foo}}([$(curmod_prefix)Bar(1), $(curmod_prefix)Bar(2)] => [$(curmod_prefix)Foo(1), $(curmod_prefix)Foo(2)])"
+    @test sprint(show, MIME("text/plain"), d) == """
```

Comment:
I think you can use the `replstr` function (defined at the top of "test/show.jl") for that

Reply:
Thanks, rerunning CI for it felt a bit inefficient so I merged as-is. But I can fix it up in the next I add in this file.

Reply:
Sure, agreed, I will also try to remember about it when I touch this file.